# Disables use of a binary for preloading the Media Engagement index
# Said binary is: chrome/browser/resources/media/mei_preload/preloaded_data.pb
# According to media/base/media_switches (for PreloadMediaEngagementData), it
# "enables a list of origins to be considered as having a high MEI until there
# is enough local data to determine the user's preferred behavior." This feature
# does not seem to outweigh the benefit of removing the binary, thus this patch.

--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -346,7 +346,6 @@ if (!is_android && !is_mac) {
     }
 
     data_deps += [
-      "//chrome/browser/resources/media/mei_preload:component",
       "//third_party/widevine/cdm",
     ]
   }
@@ -1183,7 +1182,6 @@ if (is_win) {
       ":optimization_guide_library",
       ":swiftshader_library",
       ":widevine_cdm_library",
-      "//chrome/browser/resources/media/mei_preload:component_bundle",
     ]
 
     if (is_chrome_branded) {
